client/src/main/kotlin/app/cash/backfila/client/BackfilaManagementClient.kt

```diff
@@ -14,5 +16,9 @@ interface BackfilaManagementClient {
     scan_size: Long? = null,
     pkey_range_start: ByteString? = null,
     pkey_range_end: ByteString? = null,
-  )
+  ): CreateAndStartResponse?
```

Comment:
This is just for Java and Kotlin. We don't want protos. There is no actual networking client involved.